CODEOWNERS

```diff
@@ -36,6 +36,7 @@
 /applications/nrf_desktop/                @pdunaj
 /applications/serial_lte_modem/           @junqingzou @pirun @rlubos
 /applications/zigbee_weather_station/     @adsz-nordic @tomchy
+/applications/nrf5340_audio/              @koffes @alexsven @erikrobstad @rick1082 @nordic-auko
```

Comment:
Please move this record between lines 36 and 37 (after `/applications/nrf_desktop/`) - to maintain alphabetical order.

Reply:
Corrected